### PR TITLE
[DEV-9886] Allow rails 5.2.8.1

### PR DESCRIPTION
**Pull request by ray-curran**
https://benchprep.atlassian.net/browse/DEV-9886

Allow for gem to work with Rails 5.2.8.1

### CODE REVIEW

File: onesie.gemspec [hunk 1]
```diff
@@ -42,7 +42,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop-rspec',       '~> 2.5'
   spec.add_development_dependency 'sqlite3',             '~> 1.3.13'
 
-  spec.add_runtime_dependency 'activerecord', '>= 4.2.11.3', '<= 5.2.8'
+  spec.add_runtime_dependency 'activerecord', '>= 4.2.11.3', '<= 5.2.8.1'
   spec.add_runtime_dependency 'colorize',     '~> 0.8.1'
-  spec.add_runtime_dependency 'railties',     '>= 4.2.11.3', '<= 5.2.8'
+  spec.add_runtime_dependency 'railties',     '>= 4.2.11.3', '<= 5.2.8.1'
 end
```
Comment (nick-desteffen) [inline onesie.gemspec:48]: Can we just do 5.2 and not worry about the patch versions?

Comment (ray-curran) [reply onesie.gemspec:48]: We could make it `< 5.3`. Would that work? I can't find another way to allow any version of 5.2 without removing support for 4.2.11.3 - 5.1.X

Comment (nick-desteffen) [reply onesie.gemspec:48]: Hmm, `< 6` ?

Comment (nick-desteffen) [reply onesie.gemspec:48]: I guess at the end of the day, it really doesn't matter...

**Review by nick-desteffen — APPROVED** [review]

**Pull request merged**